src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -491,39 +692,34 @@ export const WorkspaceForm = ({
                       )}
                       onChange={handleFeatureChange}
                       style={{ marginLeft: 40 }}
+                      data-test-subj={`workspaceForm-workspaceFeatureVisibility-${featureOrGroup.name}`}
```

Comment:
It seems the test id is the same as the checkbox one, do we need to use a new one?